UnityProject/Assets/Scripts/Objects/Doors/DoorMasterController.cs

```diff
@@ -157,6 +158,41 @@ public void UpdateNewPlayer(NetworkConnection playerConn)
 				ConstructibleDoor != null && ConstructibleDoor.Panelopen);
 		}
 
+		public void OnEMP(int EMPStrength = 0)
```

Comment:
You can move this to the hacking script to make it do more random things, and be more compatible with stuff when packing is added to other things

Reply:
done